scripts/updateConsent.js

```diff
@@ -0,0 +1,28 @@
+var fs = require('fs');
+const {Pool} = require('pg');
+
+function updateConsent(database){
+  //Still need to grab emails from a sensitive data file
+  const emails = JSON.parse(fs.readFileSync('./tmp/test.json'))
+  const emailString = emails.consented.map(x => "'" + x + "'").toString();
+  const pool = new Pool({database});
+  const sql = `
+    INSERT INTO consented_email (email,audio,permission,consent) 
+    SELECT x,TRUE,TRUE,TRUE 
+    FROM unnest(ARRAY[${emailString}]) x;`;
+  return pool.query(sql);
+}
+
+// Create a new database for test or development mode, and 
+// create the tables for the database.
+const databaseName = process.argv[2];
```

Comment:
This is great, scripting this is even better than adding this manually!  And also awesome how you put it as a command in the the `package.json` so folks can discover it.  One thing that might help is adding a line or two to the README about how to use this.

Also, if I read it right, this script expects the `consent-latest.json` format, which was produced by a separate script in the private repo that ran on the TSV file downloaded from Google Sheets.  So if I follow, I think for this to fully work you'll need to either check in a version of that file, or incorporate it into this script here.

Reply:
sounds good. I've added another script that is basically a replica of the one from the private repo and added extra commands in the `package.json` to use them.

The README is currently completely out of date. I'll work on that bit after this Friday's playtest!